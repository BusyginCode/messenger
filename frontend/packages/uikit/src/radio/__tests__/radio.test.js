import React from 'react'
import { render } from 'enzyme'
import Radio from '../radio'

describe('uikit/Radio', () => {
  it('renders correctly', () => {
    const wrapper = render(<Radio value="some-val" checked />)
    expect(wrapper).toMatchSnapshot()
  })
})
