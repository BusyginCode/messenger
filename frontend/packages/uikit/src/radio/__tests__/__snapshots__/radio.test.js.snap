// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/Radio renders correctly 1`] = `
<span
  class="dt-radio dt-radio-checked"
>
  <input
    checked=""
    class="dt-radio-input"
    type="radio"
    value="some-val"
  />
  <span
    class="dt-radio-inner"
  />
</span>
`;
